astropy/cosmology/_io/tests/test_json.py

```diff
@@ -20,7 +21,7 @@ def read_json(filename, **kwargs):
 
     Parameters
     ----------
-    filename : str
+    filename : str | bytes | os.PathLike
```

Comment:
New feature that needs change log?

Reply:
It's already checked for str | bytes | os.PathLike.

Reply:
So, just a doc fix of existing feature?